phpstan.neon

```diff
@@ -1,10 +1,12 @@
 includes:
+    - vendor/phpstan/phpstan-phpunit/extension.neon
+    - vendor/phpstan/phpstan-symfony/extension.neon
     - phpstan-baseline.neon
 
 parameters:
     ignoreErrors:
         - "#^Cannot call method (log|debug|info|notice|warning|error|critical|alert|emergency)\\(\\) on Psr\\\\Log\\\\LoggerInterface\\|null\\.$#"
-    level: max
+    level: 8
```

Comment:
Why was it changed? `9` is the maximum level AFAIK.

Reply:
Sure, but what changed that, level `8` instead of `max` is now the preferred one?

Reply:
All clear, to me sticking to level `8` is fine.